src/renderer/components/node/IteratorHelperNodeFooter.tsx

```diff
@@ -19,33 +20,53 @@ const NodeFooter = memo(({ validity }: NodeFooterProps) => {
             w="full"
         >
             <Spacer />
-            <Tooltip
-                hasArrow
-                borderRadius={8}
-                closeOnClick={false}
-                gutter={24}
-                label={validity.isValid ? 'Node valid' : validity.reason}
-                px={2}
-                py={1}
-                textAlign="center"
-            >
-                <Center
-                    className="nodrag"
-                    my={-1}
+            {animated ? (
```

Comment:
Do `IteratorHelperNodeFooter` and `NodeFooter` use the same code here? If so, could you please extract this out into its own component?